### ISSUE TITLE
Fix Cloud Shell telemetry so that images breakdown works correctly

### ISSUE THREAD

**Issue by simonz130** [GoogleCloudPlatform/cloud-ops-sandbox#502]
With the latest changes we did to custom cloud shell image certification, now the certified image is tagged as "latest" always. As a result, our telemetry that shows breakdown of usage by version will break and show all the versions under "latest" tag.

**Comment by arbrown** [comment]
@daniel-sanche - do you have any details on this?  This pipeline is still not working (will be fixed soon), so I'm not sure what data would be there.

**Comment by daniel-sanche** [comment]
IIRC, this issue as Simon described it is probably out of date. 

Context: for a while, we moved from our certified Cloud Shell image to a certified one, but that introduced new issues. One of them is that cloud shell images are always marked as `latest`, which was messing with our telemetry data. Another issues (that actually caused us to go back to uncertified images) was that we couldn't control the rollouts of new versions, because you can only have one "certified" image at a time, and the Cloud Shell team determines when new versions are rolled out

TL;DR, I think since we went back to uncertified images, this likely isn't an issue anymore. But I'm not very familiar with the telemetry side of things, so you might want to double-check

**Comment by minherz** [comment]
I will close this issue following [previous comment](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/issues/502#issuecomment-1211146325). We always can reopen it if we decide to investigate the problem.
